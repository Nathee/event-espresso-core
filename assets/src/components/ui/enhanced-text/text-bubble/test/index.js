/**
 * External imports
 */
import { shallow } from 'enzyme';

/**
 * Internal imports
 */
import TextBubble from '../index';

describe( 'TextBubble', () => {
	it( 'renders as expected with default props', () => {
		const wrapper = shallow( <TextBubble /> );
		expect( wrapper ).toEqual( {} );
	} );
	it( 'renders provided string content as expected', () => {
		const wrapper = shallow( <TextBubble content={ 'Test Content' }/> );
		expect( wrapper ).toMatchSnapshot();
	} );
} );