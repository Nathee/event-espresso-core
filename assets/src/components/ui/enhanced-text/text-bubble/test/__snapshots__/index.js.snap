// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TextBubble renders provided string content as expected 1`] = `
<span
  className="ee-text-bubble"
>
  Test Content
</span>
`;
